### PR TITLE
Update CVE-2021-44142.json

### PR DESCRIPTION
**Pull request by jstanley10**
This CVE ID may have been reserved by @zdi-team

### CODE REVIEW

**Comment by cve-team** [comment]
jstanley10 is not recognized currently as an account participating in the Git Pilot. To ensure timely processing of your pull request, please contact the CVE Team using either the CVE Request web form (https://cveform.mitre.org/) or an email (cve@mitre.org) and confirm your identity.

**Comment by jstanley10** [comment]
Hello @cve-team, I attempted to reply with an email to cve@mitre.org and received a response that it was an unmonitored address. @zmanion can confirm I work for CERT/CC.

**Comment by zmanion** [comment]
> Hello @cve-team, I attempted to reply with an email to [cve@mitre.org](mailto:cve@mitre.org) and received a response that it was an unmonitored address. @zmanion can confirm I work for CERT/CC.

Confirming that @jstanley10 works for the CERT/CC CNA and is authorized on our behalf to submit CVE content.

**Comment by anthnysingleton** [comment]
Hey @zmanion and @jstanley10 ,

Thanks for your patience, I will add you to the git pilot.

Best, 
Anthony

File: 2021/44xxx/CVE-2021-44142.json [hunk 1]
```diff
@@ -1,18 +1,101 @@
 {
```
Comment (cve-team) [inline 2021/44xxx/CVE-2021-44142.json:1]: CNA(s) of user (certcc) do not match owning CNA.

**Review by cve-team — CHANGES_REQUESTED** [review]
Auto validation errors:

File: 2021/44xxx/CVE-2021-44142.json (hunk 1 above)
Comment (cve-team) [inline 2021/44xxx/CVE-2021-44142.json:1]: CNA(s) of user (certcc) do not match owning CNA.

**Review by cve-team — CHANGES_REQUESTED** [review]
Auto validation errors:

**Comment by anthnysingleton** [comment]
There is an assignment issue with this request. But I also need to get your signature key so you can make future PRs without issues. Can you open a request with our webform https://cveform.mitre.org/  selecting the "Other" option and supplying your signature key?

This will also start an email thread where we can discuss the other issue with this PR as well.

**Comment by zmanion** [comment]
Hi @anthnysingleton,
For PGP, @jstanley10 can provide a key, but I've noticed some of my PRs getting a bad/unknown signature result from the CI, and then I didn't PGP sign a PR or two and they were accepted. So, just want to confirm that you would like PGP-signed commits and we'll do it.

**Comment by zmanion** [comment]
For the CNA conflict, we expected this, we are not the requesting/assigning CNA. Not sure who is, I'm guessing ZDI or Red Hat or even someone from Samba. I look forward to the future when requester/assigner is part of the JSON to save these round trips.

Not sure what the protocol is, somebody has to contact the requester/assigner and can they then allow our PR, or create their own submission in a timely manner? This has been reserved-but-public for a couple weeks which is why we chose to write it up.

We can seek that approval but don't know whom to ask.

**Comment by anthnysingleton** [comment]
Hey @zmanion,

Yes on our end we have three keys for your zmanion account. We may want to clean that up so that we can reduce those errors. We do not require you to sign your commits but some organizations prefer to sign their commits so that is an additional step in getting into the Pilot. I remember seeing you sign a previous PR which is why I asked for @jstanley10 key. I will leave it to you to decide if you want your org and its approved submitters to sign their commits.

**Comment by anthnysingleton** [comment]
> 

Yes it was assigned by RedHat. We have few options

1. Have the RedHat org "bless" this PR with a comment approving.
2. You can email Redhat/message RedHat on our Slack group where MITRE can see this agreement
3. [Official] Submit a request through the webform asking for assigning org to be changed to CERT/CC, MITRE will then reach out to RedHat for agreement.

**Comment by tausif-rh** [comment]
> > 
> 
> Yes it was assigned by RedHat. We have few options
> 
> 1. Have the RedHat org "bless" this PR with a comment approving.
> 2. You can email Redhat/message RedHat on our Slack group where MITRE can see this agreement
> 3. [Official] Submit a request through the webform asking for assigning org to be changed to CERT/CC, MITRE will then reach out to RedHat for agreement.

Red Hat: 

We see the owning CNA now shows to be certcc. I agree to approve the PR as it's public now.

**Comment by zmanion** [comment]
> I will leave it to you to decide if you want your org and its approved submitters to sign their commits.

We'd like to *not* PGP sign commits, we anticipate transition to the new services soon and sufficiently trust GitHub authentication and authorization. I can provide the list of GitHub user IDs that are authorized for the CERT/CC CNA, would you like those via the form?

**Comment by anthnysingleton** [comment]
> We see the owning CNA now shows to be certcc. I agree to approve the PR as it's public now.

I changed it temporarily to get the PR processed. Thanks for replying via email @tausif-rh 

> We'd like to not PGP sign commits, we anticipate transition to the new services soon and sufficiently trust GitHub authentication and authorization. I can provide the list of GitHub user IDs that are authorized for the CERT/CC CNA, would you like those via the form?

@zmanion Yes if there are more users contributing to the GitHub Pilot please submit their information via the web form.

**Pull request merged**